### ISSUE TITLE
Entities with unusually high health causing high volume of lines to be sent through ACT in some cases.

### ISSUE THREAD

**Issue by Zaichata** [ravahn/FFXIV_ACT_Plugin#249]
There are some entities within FFXIV that have a health pool above their given maximum, which if "include hp for triggers" is enabled has the potential to cause a high volume of lines to be sent through ACT at once if their HP is suddenly set back down to 100% or another arbitrary value.
An example would be in Eden's Verse: Iconoclasm and its savage version. There are three entities which spawn once entering and over 30 seconds progressively get more health until they are at 28,738,640 / 148,000 hp. A wipe will cause these three to despawn and respawn, which causes no issues, but upon killing the boss and clearing the instance the three entities will set their health to 148,000 / 148,000 hp before despawning which causes the plugin to count down from 19418% to 100% sending almost 60,000 lines at once.

An example of one of the three entities within Eden's Verse: Iconoclasm:
`[02:11:57.687] 03:4000A219:Added new combatant .  Job: N/A Level: 80 Max HP: 148000 Max MP: 10000 Pos: (100,70,2.384186E-07) (10800000011579).`
`[02:12:03.355] 27:4000A219::3021864:148000:0:10000:0:0:100:70:2.384186E-07:-4.792213E-05:`
`[02:12:06.359] 27:4000A219::5895728:148000:0:10000:0:0:100:70:2.384186E-07:-4.792213E-05:`
`[02:12:09.317] 27:4000A219::8769592:148000:0:10000:0:0:100:70:2.384186E-07:-4.792213E-05:`
`[02:12:12.318] 27:4000A219::11643456:148000:0:10000:0:0:100:70:2.384186E-07:-4.792213E-05:`
`[02:12:15.321] 27:4000A219::14517320:148000:0:10000:0:0:100:70:2.384186E-07:-4.792213E-05:`
`[02:12:18.325] 27:4000A219::17391184:148000:0:10000:0:0:100:70:2.384186E-07:-4.792213E-05:`
`[02:12:21.327] 27:4000A219::20265048:148000:0:10000:0:0:100:70:2.384186E-07:-4.792213E-05:`
`[02:12:24.329] 27:4000A219::23138912:148000:0:10000:0:0:100:70:2.384186E-07:-4.792213E-05:`
`[02:12:27.326] 27:4000A219::26012776:148000:0:10000:0:0:100:70:2.384186E-07:-4.792213E-05:`
`[02:12:30.326] 27:4000A219::28738640:148000:0:10000:0:0:100:70:2.384186E-07:-4.792213E-05:`
`[02:20:50.764] 22:4000A219::4000A219::00`
`[02:20:51.342] 27:4000A219::148000:148000:0:10000:0:0:100:70:0:-4.792213E-05:`
`[02:21:00.316] 04:4000A219:Removing combatant .  Max HP: 1343488. Pos: (100,70,0).`

**Comment by ravahn** [comment]
Yes, I see how that could be happening in the code.  I will update this to not report percents > 100%.
